src/pages/hosts/RestartJasper.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
```

Comment:
since this component is shared between multiple pages it should go in the components directory alongside the update status modal